src/applications/vaos/reducers/index.js

```diff
@@ -1,5 +1,10 @@
+import { combineReducers } from 'redux';
+import appointments from './appointments';
 import newAppointment from './newAppointment';
 
 export default {
-  newAppointment,
+  vaos: combineReducers({
```

Comment:
I think we'll need to adjust the couple form pages we created to add this path.

Reply:
I removed the `combinedReducers` for a flatter structure like you suggested so that should resolve this.